apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -119,4 +120,18 @@ test.describe("pro user", () => {
     await page.goto(`/${pro.username}`);
     await bookFirstEvent(page);
   });
+
+  test("can book an event that requires confirmation and then that booking can be accepted by organizer", async ({
+    page,
+    users,
+  }) => {
+    await bookOptInEvent(page);
+    const [pro] = users.get();
+    await pro.login();
+
+    await page.goto("/bookings/unconfirmed");
+    await page.click('[data-testid="confirm"]');
+    // This is the only booking in there that needed confirmation and now it should be empty screen
+    await page.waitForSelector('[data-testid="empty-screen"]');
```

Comment:
Shouldn't test include expects to validate it works?

Reply:
It's a kind of indirect assertion, to verify the presence of an element.

Reply:
Let's add an explicit expect so we don't actually wait for the timeout and fail fast.

Reply:
Sorry, I remember you fixing a lot of similar issues in my tests earlier. Fixed it now